CHANGELOG.rst

```diff
@@ -6,6 +6,13 @@ Changelog
 Releases
 --------
 
+v1.1.0 (2019-07-13)
+~~~~~~~~~~~~~~~~~~~
+
+* Added `jobflow_role` CLI argument to configure cluster EC2 Instance Profile
```

Comment:
Let's change it to present e.g. `Add `jobflow_role` CLI`

Reply:
Also updated the rest of the changelog items to present tense